core/vpnlauncher/src/main/java/com/kape/vpnlauncher/VpnLauncher.kt

```diff
@@ -38,7 +38,7 @@ class VpnLauncher(
                 return
             } else {
                 val server: VpnServer = connectionPrefs.getSelectedVpnServer()
-                    ?: regionListProvider.servers.value.first { it.autoRegion }
+                    ?: regionListProvider.getAutoServer()
```

Comment:
shouldn't this be `getOptimalServer()`? Otherwise it looks like it would pick always a random one

Reply:
You're right! Updated.